frontend/src/hooks/secret-operations/useCreatePersonalSecret.ts

```diff
@@ -0,0 +1,46 @@
+import { useCallback } from "react";
+
+import { useFetchSecretValue } from "../api/dashboard/queries";
+import { SecretType } from "../api/types";
+import { useHandleSecretOperation } from "./useHandleSecretOperation";
+
+interface CreatePersonalSecretParams {
+  key: string;
+  secretPath: string;
+  environment: string;
+}
+
+export function useCreatePersonalSecretOverride(projectId: string) {
```

Comment:
Since personal override as of this change can be created from the overview and from the secret list page, I pulled the logic here to share.

This is the only logic about personal overrides that should be in those component. Once created, they render `SecretPersonalOverrideView` and forget.